src/components/CommentCard.jsx

```diff
@@ -1,10 +1,35 @@
 /* eslint-disable react/prop-types */
+import { useContext, useState } from "react"
+import { UserContext } from "../contexts/User"
+import { deleteComment } from "../api"
+
+
+function CommentCard({comment, onDelete, canDelete, setComments}){
+    const {user} =useContext(UserContext)
+    const [isDeleting, setIsDeleting] = useState(false)
+
+    function handleDelete(){
+        console.log(comment.comment_id, '<<<deleting comment with ID')
+        setIsDeleting(true)
+        deleteComment(comment.comment_id)
+        .then(() => {
+            setComments((prevComments) => prevComments.filter(oldComments => oldComments.comment_id !==comment.comment_id))
+            console.log('comment deleted successfully')
+            // onDelete(comment.comment_id)
+            setIsDeleting(false)
+        })
+        .catch((error) => {
+            console.error('failed to delete comment. try again later', error)
+        })
```

Comment:
As it stands, there is no feedback to the user that the deleting comment has failed. Instead of console logging it, you should let the user know on the frond end instead. Please do add this in when you get a chance